README.md

```diff
@@ -71,6 +74,11 @@ cd ros2_latest_ros_launch_sandbox_ws
 # Use vcs to clone all required repositories
 curl https://raw.githubusercontent.com/ros2/ros2/dashing/ros2.repos | vcs import src/
 curl https://raw.githubusercontent.com/aws-robotics/launch-ros-sandbox/master/launch_ros_sandbox.repos | vcs import src/
+# Install all required system dependencies
+# Some packages may fail to install, this is expected on an unstable branch,
+# and is generally OK.
+rosdep update
+rosdep install -r --rosdistro=eloquent --ignore-packages-from-source --from-paths src/
```

Comment:
Dashing for consistency?

Reply:
This one is master, so it needs to target the next release. The dashing instruction do not incorporate a `--rosdistro` flag because in this case, it is "guessed" when you source `/opt/ros/dashing/setup.sh` which defines the ROSDISTRO environment variable to 'dashing'. For a full source build, 'ROSDISTRO' is unset and so we need to set it manually.